src/gather/api.py

```diff
@@ -125,40 +141,46 @@ def callback(scanner, inner_name, objct):
                 effective_name = name
             objct = transform(objct)
             scanner.update(effective_name, objct)
+
         def attach(func):
             """Attach callback to be called when object is scanned"""
             venusian.attach(func, callback, depth=self.depth)
             return func
         return attach
 
     def collect(self, strategy=one_of.__func__):
-        """Collect all registered.
+        """
+        Collect all registered.
```

Comment:
I'm changing this to "functions or classes". Technically classes are callable (sometimes -- it is possible to make a class non-callable with the right metaclass), but venusian cannot register arbitrary callables.